posts/2021-01-22-ee9-messaging-security-21002-beta.adoc

```diff
@@ -0,0 +1,211 @@
+---
+= Jakarta EE 9 Messaging and Security packages now available with Open Liberty 21.0.0.2-beta
+Austin Bailey <https://github.com/austin0>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+New in Open Liberty 21.0.0.2-beta are four new Jakarta EE 9 packages, Jakarta Messaging 3.0, Jakarta Security 2.0, Jakarta Batch 2.0, and Jakarta XML Web Services 3.0,that join the large stable of Jakarta EE 9 packages already available within Open Liberty. Open Liberty 21.0.0.2-beta also includes previews for 8 of the updated MicroProfile modules, including Config, Metrics, OpenTracing, Fault Tolerance, Health, OpenAPI, Rest Client, and JWT. 
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* link:https://openliberty.io/blog/2020/10/27/microprofile40-jakartaee9-beta-200012.html#conf[MicroProfile Config 2.0]
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#metrics[MicroProfile Metrics 3.0]
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#opentracing[MicroProfile OpenTracing 2.0]
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#fault[MicroProfile Fault Tolerance 3.0]
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#health[MicroProfile Health 3.0]
+* link:https://openliberty.io/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[MicroProfile OpenAPI 2.0]
+* link:https://openliberty.io/blog/2020/11/24/microprofile-rest-client-kubernetes-secrets-21001beta.html#MP[MicroProfile Rest Client 2.0]
+* link:https://openliberty.io/blog/2020/10/27/microprofile40-jakartaee9-beta-200012.html#jwt[MicroProfile JWT 1.2]
+* link:https://openliberty.io/blog/2020/11/24/microprofile-rest-client-kubernetes-secrets-21001beta.html#kube[Using Kubernetes secrets in Liberty variables]
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.2-beta</version>
+  <type>pom</type>
+</dependency>
+----
```

Comment:
I think this needs updating a bit:
```suggestion
=== Try it now 

To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.

If you're using link:{url-prefix}/guides/maven-intro.html[Maven], add the following `runtimeArtifact` section to the `configuration` section of your `pom.xml` file:

[source,xml]
----
<runtimeArtifact>
    <groupId>io.openliberty.beta</groupId>
    <artifactId>openliberty-runtime</artifactId>
    <version>21.0.0.2-beta</version>
    <type>zip</type>
</runtimeArtifact>
----
```

Reply:
Updated, thanks.

Reply:
This was updated in one place, but not the other one completely.

Reply:
Updated for EE9 section.